CorsixTH/Lua/research_department.lua

```diff
@@ -326,79 +326,71 @@ function ResearchDepartment:getResearchRequired(thing)
   return required
 end
 
---[[ Add some more research points to research progress. If
-autopsy_room is specified points are not used. Instead research
-progresses according to the level config for autopsies.
-Otherwise they will be divided according to the research policy
-into the different research areas.
-!param points (integer) The total amount of points (before applying
-any level specific divisors to add to research.
-!param autopsy_room (string) If a specific room should get points following
-an autopsy, then this is the id of that room.
-]]
-function ResearchDepartment:addResearchPoints(points, autopsy_room)
-
+--! This function will add progress to discovering a room/object to treat a diagnosed patient after an autopsy.
+--! Research points are added as according to the level config.
+--!param target_room_id (string) The name of the room
```

Comment:
Something like "Id of the room that treats the illness, target for getting research points." ?
(yes, an id is also a name in this program), but that's generally not the case in other programs (IDs are usually numbers rather than strings).

Line 330 seems better fitting around line 342.  This text describes the call interface (ie what information to provide and what does it do and return as result). Points are not in that interface, so not highly essential to mention here.